packages/cli/package.json

```diff
@@ -31,6 +31,7 @@
     "@react-native-community/cli-server-api": "14.0.1",
     "@react-native-community/cli-tools": "14.0.1",
     "@react-native-community/cli-types": "14.0.1",
+    "@react-native-community/cli-package-manager": "14.0.1",
```

Comment:
not sure if we want to include this package inside main `package.json`. wdyt @thymikee?

Reply:
we need it as it's used inside `packages/cli/**`, same we need it in tsconfig.json (and it's there)